docs/examples/800_istio.adoc

```diff
@@ -0,0 +1,169 @@
+----
+
+Then a Coherence*Extend client can connect to the proxy server via TLS protocol at port 8043.
+
+== Using the Coherence operator with Istio
+
+NOTE: These instructions assume that you are using a Kubernetes cluster with Istio installed and configured already.
+
+You can deploy the operator into a namespace which has Istio automatic sidecar injection enabled. Before installing the operator, create the namespace in which you want to run the Coherence operator and label it.
+
+[source,bash]
+----
+kubectl create namespace coherence-operator
+----
+
+Istio Sidecar AutoInjection is done automatically if you label coherence-operator namespace with istio-injection. If you want to implement manual injection with the deployments then you need to use istioctl kube-inject.
+
+[source,bash]
+----
+kubectl label namespace coherence-operator istio-injection=enabled
+----
+
+After the namespace is labeled, you can install the operator using your preferred method in the Operator https://oracle.github.io/coherence-operator/docs/latest/#/installation/01_installation[Installation Guide].
+
+After installed operator, use the following command to confirm the operator is running:
+
+[source,bash]
+----
+kubectl get pods -n coherence-operator
+----
+
+== Creating a Coherence cluster with Istio support
+
+You can configure your cluster to run with Istio automatic sidecar injection enabled. Before creating your cluster, create the namespace in which you want to run the cluster and label it for automatic injection.
+
+[source,bash]
+----
+kubectl create namespace coherence-cluster
+kubectl label namespace coherence-cluster istio-injection=enabled
+----
+
+There is no other requirements to run Coherence in Istio environment.
+
+The following is an example that creates a cluster named example-cluster:
+
+example.yaml
+[source,bash]
+----
+# Example
+apiVersion: coherence.oracle.com/v1
+kind: Coherence
+metadata:
+  name: example-cluster
+----
+
+[source,bash]
+----
+$ kubectl -n coherence-cluster apply -f example.yaml
+----
+
+After you installed the Coherence cluster, run the following command to view the pods:
+
+[source,bash]
+----
+$ kubectl -n coherence-cluster get pods
+
+NAME                                             READY   STATUS    RESTARTS   AGE
+example-cluster-0                                2/2     Running   0          45m
+example-cluster-1                                2/2     Running   0          45m
+example-cluster-2                                2/2     Running   0          45m
+----
+
+You can see that 3 members in the cluster are running with 3 pods. 2/2 in READY column means that there are 2 containers running in each Pod. One is Coherence member and the other is Envoy Proxy.
+
+== Prometheus
+
+You can use Prometheus with Istio to record metrics that track the health of Istio and of Coherence cluster within the service mesh and view them through Granfana. 
```

Comment:
That is a bit of an open ended statement. Presumably Istio add additional metrics into Prometheus that are scraped by Prometheus from some Istio endpoint somewhere. Can you see these in Grafana? Is it easy to link these metrics back to specific Coherence clusters and Pods? As a later task - if it is not easy to link information back to Coherence clusters is there something we can do in our Grafana dashboards to integrate Istio Metrics into Coherence dashboards - which might be cool.